gradle/publication.gradle

```diff
@@ -1,20 +1,31 @@
 apply plugin: 'maven-publish'
 apply plugin: 'signing'
 
-android {
-    publishing {
-        singleVariant('release') {
```

Comment:
Can you name it `'java'`, so we do not need to add another split in `publishing` block?

Reply:
@CherryPerry I don't believe so. This is the `release` variant of the library as opposed to `debug`. If I named it `java` it wouldn't find the build.

Also, I am improving the plugin in this PR: https://github.com/bumble-tech/appyx/pull/51